tests/integration/coordinator/plugins/clickhouse/test_plugin.py

```diff
@@ -188,7 +188,6 @@ async def setup_cluster_content(clients: Sequence[HttpClickHouseClient], use_nam
     await clients[0].execute(
         b"CREATE TABLE default.mysql (a Int) ENGINE = MySQL('https://host:1234', 'database', 'table', 'user', 'password')"
     )
-    await clients[0].execute(b"CREATE TABLE default.s3 (a Int) ENGINE = S3('http://bucket.s3.amazonaws.com/key.json')")
```

Comment:
Can you make that dependent on the presence of the engine instead of disabling it ?

(By using `SELECT name FROM system.table_engines`)

Reply:
I have fixed this.
But I have added incomplete set of table engines and did not used it in `Table`. Is it fine? (not sure if enum should not be used in `Table` because some new engines can be easily added to Clickhouse)